samples/src/commonMain/kotlin/ru/nsk/samples/ComplexSyntaxSample.kt

```diff
@@ -20,7 +20,7 @@ private object ComplexSyntaxSample {
 fun main() = runBlocking {
     val machine = createStateMachine(
         this, // coroutine scope used for this machine
-        "Traffic lights" // StateMachine name is optional
+        name = "Traffic lights" // StateMachine name is optional
```

Comment:
is this neccessary? it does not compile without this line, or its just cosmetic change?